packages/components/src/button/style.scss

```diff
@@ -288,14 +288,15 @@
 	}
 
 	&.is-small {
-		height: 24px;
+		height: $icon-size;
 		line-height: 22px;
-		padding: 0 8px;
+		padding: 0;
```

Comment:
Did you mean to remove the horizontal padding here? I noticed this affects the reset button for the `BoxControl`.

| Before | After |
|---|---|
| <img width="277" alt="Screen Shot 2022-09-05 at 6 06 57 pm" src="https://user-images.githubusercontent.com/60436221/188402338-a7da89e1-6473-4f5f-84cd-751cf0d56f51.png"> | <img width="263" alt="Screen Shot 2022-09-05 at 6 17 49 pm" src="https://user-images.githubusercontent.com/60436221/188402377-8cd99783-305e-4616-91b9-aa36dd0a0854.png"> |

Reply:
This also breaks other `isSmall` buttons in the editor such as those for the image block's sizes etc.

<img width="273" alt="Screen Shot 2022-09-05 at 6 19 58 pm" src="https://user-images.githubusercontent.com/60436221/188402867-e7f8aca2-52dc-4448-9078-3e245830408c.png">

Reply:
Thank you for excellent feedback and thank you for looking. It does seem like this PR is unlikely to land as is.

With this PR in mind as an example, is there a different and safer approach we can take, to ensure the link/unlink button can move forward, i.e. componentization? Or do you still see the iterative approach as taken in this PR as potentially feasible?

Reply:
Cool thank you, I'll see if I can polish this along as I get a breather! 🙏